subprojects/android-test/test-report/src/test/kotlin/com/avito/android/test/report/transport/DataSetSerializerKtTest.kt

```diff
@@ -21,4 +19,11 @@ class DataSetSerializerKtTest : PreTransportMappers {
         assertThat(SinglePropertyDataSet("myValue").serialize())
             .isEqualTo(mapOf("myProperty" to "myValue"))
     }
+
+    @Test
+    fun `serialize() - not serializing lambda`() {
+        class LambdaPropertyDataSet(@Suppress("unused") val myLambda: () -> String) : DataSet
+
+        assertThat(LambdaPropertyDataSet { "Hello" }.serialize()).isEqualTo(emptyMap<String, Any>())
```

Comment:
It'd be helpful to see in a report a self-described value "Unsupported type: lambda value <issue>"